platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/util/BootstrapUtils.java

```diff
@@ -89,20 +90,22 @@ public final class BootstrapUtils {
     private BootstrapUtils() {}
 
     /**
-     * Load the configuration for the platform.
+     * Load settings from a legacy settings.txt source.
      *
-     * @param configurationBuilder the configuration builder to setup
-     * @param settingsPath         the path to the settings.txt file
-     * @throws IOException if there is a problem reading the configuration files
+     * @param configurationBuilder the configuration builder
+     * @param settingsPath         the path to the settings.txt file (does not actually have to be named
+     *                             "settings.txt")
      */
-    public static void setupConfigBuilder(
-            @NonNull final ConfigurationBuilder configurationBuilder, @NonNull final Path settingsPath)
-            throws IOException {
-
-        final ConfigSource settingsConfigSource = LegacyFileConfigSource.ofSettingsFile(settingsPath);
-        final ConfigSource mappedSettingsConfigSource = ConfigMappings.addConfigMapping(settingsConfigSource);
+    public static void readSettingsDotTxt(
```

Comment:
considering this doesn't have to be named settings.txt, will it be helpful to not name it readSettingsDotTxt? Maybe we name it readLegacySettings instead?

Reply:
Renamed to `readLegacySettingsFile()`